src/app/core/auth/auth.service.ts

```diff
@@ -6,6 +6,10 @@ import { LocalStorageService } from 'ngx-webstorage';
 import { map, tap } from 'rxjs/operators';
 import { CartService } from '../cart/cart.service';
 import { CurrentUserService } from './current-user.service';
+import { getApiUrl } from '../../utils/helpers';
+import { ServiceAlias } from '../../types/services';
+
+const apiUrl = getApiUrl(ServiceAlias.Auth);
```

Comment:
Would be better if it is part of the class, not defined globally. 
In constructor you can define it: ```this.apiUrl = getApiUrl(ServiceAlias.Auth)``` and then use ```this.apIUrl```.